src/components/App/App.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './App.scss';
+import { Sum } from '../Sum';
+
+export const App = () => (
+  <>
```

Comment:
Unnecessary fragment here. On top levels better to add wrappers.
```suggestion
  <div>
```